httomo/runner/task_runner.py

```diff
@@ -8,6 +8,8 @@
 from mpi4py import MPI
 
 import httomo
+from httomo.globals import *
```

Comment:
This is personal preference, so feel free to take it or leave it.

I tend to prefer avoiding `*` imports. I can see that `run_out_dir` and `MAX_CPU_SLICES` are coming from `httomo.globals`, but I personally prefer to know where they're coming from explicitly (as things were before), especially if they're global variables.

Seeing `httomo.globals.run_out_dir` makes it somewhat clear to me how the task runner has gotten ahold of the output dir, but seeing just `run_out_dir` makes me wonder how/why the task runner has the output dir value floating around.

Reply:
And actually, perhaps a more serious issue with this is that I'm unsure if the global variables imported via the `*` would be updated from the perspective of the task runner after the import is done. Ie, I'm not sure if `run_out_dir` and `MAX_CPU_SLICES` being updated wherever they get updated will be reflected in the task runner, does this work if a pipeline is executed? Does `run_out_dir` get seen as the value it's updated to in `cli.py`, or is it still `None` when the task runner starts doing things?

Reply:
this change was actually because of the tests failing due to `httomo.globals` couldn't be located because of  `import httomo`. 
So I've changed to `import httomo.globals`, seems to work now.